spec/features/user_creates_snippet_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+require "spec_helper"
```

Comment:
You don't need to require `spec_helper` ~ it's already required by `rails_helper`